### PR TITLE
SectionNav refactor to ChakraUI

### PR DESCRIPTION
**Pull request by Master7130**
## Description

Converted SectionNav.tsx from styled-components to ChakraUI

## Related Issue

#6374

### CODE REVIEW

**Comment by Master7130** [comment]
The letter spacing for the heading was previously 0.04rem but ChakraUI doesn't have a default value for that. The closest one was "wider" which is 0.05rem. Is that close enough or should I manually set it to 0.04rem?

File: src/components/SectionNav.tsx [hunk 1]
```diff
@@ -1,50 +1,41 @@
...
   children?: React.ReactNode
 }
 
 const SectionNav: React.FC<IProps> = ({ children }) => (
-  <Card>
-    <TopContent>
-      <Emoji text=":point_right:" mb={`1em`} />
-      <Title>
+  <Flex
+    display="flex"
```
Comment (pettinarip) [inline src/components/SectionNav.tsx:12]:
```suggestion
```

File: src/components/SectionNav.tsx [hunk 2]
```diff
@@ -1,50 +1,41 @@
...
+    flexDirection="column"
```
Comment (pettinarip) [inline src/components/SectionNav.tsx:13]:
```suggestion
    direction="column"
```

File: src/components/SectionNav.tsx [hunk 3]
```diff
@@ -1,50 +1,41 @@
...
+    color="textTableOfContents"
+    marginTop={12}
+    justifyContent="space-between"
```
Comment (pettinarip) [inline src/components/SectionNav.tsx:16]:
```suggestion
    justify="space-between"
```

File: src/components/SectionNav.tsx [hunk 4]
```diff
@@ -1,50 +1,41 @@
...
+    background="searchBackground"
+    borderRadius="2xl"
```
Comment (pettinarip) [inline src/components/SectionNav.tsx:18]:
```suggestion
    borderRadius="base"
```

File: src/components/SectionNav.tsx [hunk 5]
```diff
@@ -1,50 +1,41 @@
...
+    border="1px solid"
+    borderColor="lightBorder"
+    padding={6}
+    paddingBottom={0}
+  >
+    <Flex display="flex" alignItems="flex-start">
```
Comment (pettinarip) [inline src/components/SectionNav.tsx:24]:
```suggestion
    <Flex align="flex-start">
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
Good job @Master7130. Left a few comments 👍🏼

**Comment by Master7130** [comment]
I noticed that some of your suggestions were to change the style names from longer names like justifyContent to justify. Is using the shorthand form a best practice?

**Comment by pettinarip** [comment]
> I noticed that some of your suggestions were to change the style names from longer names like justifyContent to justify. Is using the shorthand form a best practice?

Good question. It is (at least try) to maintain as much consistency as possible in the code base. Since by using [`Flex`](https://chakra-ui.com/docs/components/flex/usage#usage) you benefit from these shorthand props, we tend to use them to keep it more concise.

**Comment by Master7130** [comment]
@pettinarip The build is failing here but it's succeeding locally. Can you rerun the build check?

**Review by pettinarip — APPROVED** [review]
🚀

**Pull request merged**